src/lib/event_mapper/impl/general_aws_event_mapper.py

```diff
@@ -125,3 +135,24 @@ def to_message(self) -> dict:
         }
 
         return message
+
+
+class ExecutionInfoUrlMixin:
+    @staticmethod
+    def get_url(
+        resource_type: str,
+        region_name: str,
+        resource_name: str,
+        account_id: str = None,
+        run_id: str = None,
+    ) -> str:
+        """Returns the link to the particular resource run."""
+        url_mapping = {
+            types.GLUE_JOBS: f"https://{region_name}.console.aws.amazon.com/gluestudio/home?region={region_name}#/job/{resource_name}/run/{run_id}",
+            types.STEP_FUNCTIONS: f"https://{region_name}.console.aws.amazon.com/states/home?region={region_name}#/v2/executions/details/arn:aws:states:{region_name}:{account_id}:execution:{resource_name}:{run_id}",
+            types.LAMBDA_FUNCTIONS: f"https://{region_name}.console.aws.amazon.com/cloudwatch/home?region={region_name}#logsV2:log-groups/log-group/$252Faws$252Flambda$252F{resource_name}/log-events/",
```

Comment:
I'm not sure - do we handle custom log group names?
I suppose url here includes log group name

Reply:
If not - disregard the comment :)

Reply:
good point :) I think I can re-use the get_log_group method from LambdaManager so to handle custom log groups as well

Reply:
just looks like we would need to assign the permissions lambda:GetFunction and lambda:GetFunctionConfiguration for Digest and Alerting Lambdas additionally, I'll test it

Reply:
as discussed, we will consider this enhancement in the next version
at the moment, we will use the default log group